website/source/docs/builders/googlecompute.html.md

```diff
@@ -125,8 +125,11 @@ builder.
 -   `project_id` (string) - The project ID that will be used to launch instances
     and store images.
 
--   `source_image` (string) - The source image to use to create the new
-    image from. Example: `"debian-7-wheezy-v20150127"`
+-   `source_image` or `source_image_family` (string) - The source image or
+    family to use to create the new image from. The image family always returns
+    its latest image that is not deprecated. If both `source_image` and
+    `source_image_family` are specified, `source_image` takes precedence.
+    Example: `"debian-7-wheezy-v20150127"`, `"debian-7"`.
```

Comment:
I added another entry first. Then I noticed that they don't clearly separate into required and optional anymore, so I documented them together.

Do you still want two entries? I'd add `source_image_family` in the optional section and mention it up here. I would leave `source_image` up here, even though it's not strictly required anymore.